src/test/scala/rx/lang/scala/CompletenessTest.scala

```diff
@@ -1,494 +0,0 @@
-    methodMembersToMethodStrings(tp.declarations.filter {
-      m =>
-        m.isMethod && m.isPublic &&
-          m.annotations.forall(_.toString != "java.lang.Deprecated") // don't check deprecated classes
-    })
-    // TODO how can we filter out instance methods which were put into companion because
-    // of extends AnyVal in a way which does not depend on implementation-chosen name '$extension'?
-    .filter(! _.contains("$extension"))
-    // `access$000` is public. How to distinguish it from others without hard-code?
-    .filter(! _.contains("access$000"))
-  }
-
-  // also applicable for Java types
-  def getPublicInstanceAndCompanionMethods(tp: Type): Iterable[String] =
-    getPublicInstanceMethods(tp) ++
-      getPublicInstanceMethods(tp.typeSymbol.companionSymbol.typeSignature)
-
-  def printMethodSet(title: String, tp: Type) {
-    println("\n" + title)
-    println(title.map(_ => '-') + "\n")
-    getPublicInstanceMethods(tp).toList.sorted.foreach(println(_))
-  }
-
-  @Ignore // because spams output
-  @Test def printJavaInstanceMethods(): Unit = {
-    printMethodSet("Instance methods of rx.Observable",
-                   typeOf[rx.Observable[_]])
-  }
-
-  @Ignore // because spams output
-  @Test def printScalaInstanceMethods(): Unit = {
-    printMethodSet("Instance methods of rx.lang.scala.Observable",
-                   typeOf[rx.lang.scala.Observable[_]])
-  }
-
-  @Ignore // because spams output
-  @Test def printJavaStaticMethods(): Unit = {
-    printMethodSet("Static methods of rx.Observable",
-                   typeOf[rx.Observable[_]].typeSymbol.companionSymbol.typeSignature)
-  }
-
-  @Ignore // because spams output
-  @Test def printScalaCompanionMethods(): Unit = {
-    printMethodSet("Companion methods of rx.lang.scala.Observable",
-                   typeOf[rx.lang.scala.Observable.type])
-  }
-
-  def javaMethodSignatureToScala(s: String): String = {
-    s.replaceAllLiterally("Long, TimeUnit", "Duration")
-     .replaceAll("Action0", "() => Unit")
-     // nested [] can't be parsed with regex, so these will have to be added manually
-     .replaceAll("Action1\\[([^]]*)\\]", "$1 => Unit")
-     .replaceAll("Action2\\[([^]]*), ([^]]*)\\]", "($1, $2) => Unit")
-     .replaceAll("Func0\\[([^]]*)\\]", "() => $1")
-     .replaceAll("Func1\\[([^]]*), ([^]]*)\\]", "$1 => $2")
-     .replaceAll("Func2\\[([^]]*), ([^]]*), ([^]]*)\\]", "($1, $2) => $3")
-     .replaceAllLiterally("_ <: ", "")
-     .replaceAllLiterally("_ >: ", "")
-     .replaceAll("(\\w+)\\(\\)", "$1")
-  }
-
-  @Ignore // because spams output
-  @Test def printDefaultMethodCorrespondence(): Unit = {
-    println("\nDefault Method Correspondence")
-    println(  "-----------------------------\n")
-    val c = SortedMap(defaultMethodCorrespondence.toSeq : _*)
-    val len = c.keys.map(_.length).max + 2
-    for ((javaM, scalaM) <- c) {
-      println(s"""      %-${len}s -> %s,""".format("\"" + javaM + "\"", "\"" + scalaM + "\""))
-    }
-  }
```

Comment:
Several methods (including this one) are deleted here but not added to anywhere. I would like to keep them, because they might be useful for maintenance in the future.